### ISSUE TITLE
position of items after pull or load

### ISSUE THREAD

**Issue by francisnnumbi** [xuelongqy/flutter_easyrefresh#806]
Hey,

I have two scenario  for a list loaded from paginated request:

1. When I pull down the list view, I want to view the previous page items
2. When I pull up the list view, I want to view the next page items.

All is well so far.

- But when I pull down, and the page refreshes, what is displayed is the first item in the list. This should not be the case as coming from the bottom, the list view for that new page should display the last item at the bottom and then I can move up until I reach the first item and pull down again !
- The same opposite behavior when I pull up, and the page refreshes, what is displayed is the last item in the list. This should not be the case as coming from the top, the list view for that new page should display the first item at the top and then I can move down until I reach the last item and pull up again !

In short :

- Pull down, view last item of the previous page,
- Pull up, view the first item of the next page.

QUESTION
How do I achieve that ?
